src/pages/HomeParticipant.vue

```diff
@@ -375,23 +376,28 @@ const studentInfo = computed(() => {
   };
 });
 
-watch(consentParams, (newValue) => {
-  consentParams.value = newValue;
-});
+watch(
+  consentParams,
+  (newValue) => {
+    consentParams.value = newValue;
```

Comment:
I think I'm a bit confused what this watcher statement does. If the newValue of consentParam is coming in as `3`, why do we need to re-set consentParams to `3`?

Reply:
Yes, you are right, I don't need this watcher.  I had it for control before, thank you for noticing :)